public/app/plugins/datasource/cloudwatch/specs/datasource.test.ts

```diff
@@ -511,11 +511,11 @@ describe('CloudWatchDatasource', () => {
       describe('and region param is left out', () => {
         it('should use the default region', async () => {
           const { ds, instanceSettings } = getTestContext();
-          ds.doMetricQueryRequest = jest.fn().mockResolvedValue([]);
+          ds.doMetricResourceRequest = jest.fn().mockResolvedValue([]);
 
           await ds.metricFindQuery('metrics(testNamespace)');
```

Comment:
I believe you can now remove the metricFindQuery method from `datasource.ts`? I don't see it being used anywhere.

Reply:
done